### PR TITLE
Chapter1 homework 1and2

### PR DESCRIPTION
**Pull request by mertefeisikgor**
MY second homework try for 1 and 2 lessons.

### CODE REVIEW

**Review by Almantask — CHANGES_REQUESTED** [review]
I see a lot of noise in this homework , could you please clean it up?
You have a lot of files that are not ignored, double submission for solution. Have you forked the repository?

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -4,9 +4,26 @@ namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        static void Main(string[] args) 
         {
-            Console.WriteLine("Hello World!");
+            string firstName = "Mert Efe";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: PLease use console to read input. Do not hardcode input, the point was not that.

**Review by Almantask — COMMENTED** [review]
Refer to console.ReadLine and int.Parse float.Parse

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (mertefeisikgor) [reply Src/BootCamp.Chapter/Program.cs]: Hi kai I did made the necessary adjustments. Hope they are ok now.

**Review by jdphenix — APPROVED** [review]
Good job with the changes. 

One nitpick - consider commit names that make a bit more sense. You'll thank yourself later, I promise :)

**Pull request closed without merging**